okhttp/src/jvmMain/kotlin/okhttp3/Cookie.kt

```diff
@@ -93,7 +93,23 @@ class Cookie private constructor(
    * **only** `example.com`. If this flag is false it matches `example.com` and all subdomains
    * including `api.example.com`, `www.example.com`, and `beta.api.example.com`.
    */
-  @get:JvmName("hostOnly") val hostOnly: Boolean // True unless 'domain' is present.
+  @get:JvmName("hostOnly") val hostOnly: Boolean, // True unless 'domain' is present.
+
+  /**
+   * Returns a non-empty string with the cookie's policy for whether the cookie is sent with
+   * cross-site requests, providing some protection against cross-site request forgery (CSRF) attacks.
+   * Same-site requests means requests originating from the same site that set the cookie. Cross-site
+   * means requests originating from site that's different from the one that set the cookie.
+   *
+   * The possible values are:
+   * - `Strict`: Cookie will only be sent for same-site requests. The scheme and domain
+   *     must match the one on the cookie.
+   * - `Lax`: Cookie is not sent on cross-site requests. Cookie won't be sent when going from origin
+   *     site to an external site, but is sent when going from an external site to the origin site.
+   * - `None`: Browser will send the cookie with both cross-site and same-site requests. The Secure
+   *     attribute must be set when setting this value.
+   */
+  @get:JvmName("sameSite") val sameSite: String,
```

Comment:
Is String the right type for this?  It may be is we want to handle even invalid values.  But figured worth asking.

Reply:
Closing based on other comments.